Packs/Sixgill-Darkfeed/Integrations/Sixgill_Darkfeed/Sixgill_Darkfeed.yml

```
@@ -11,6 +11,17 @@ configuration:
   name: client_secret
   required: true
   type: 4
+- defaultvalue: 'all - Ingest all IOCs'
+  display: Sixgill Confidence
+  name: confidence
+  required: false
+  type: 15
+  options:
+  - all - Ingest all IOCs
+  - 90 - IOC exactly fits description
+  - 80 - IOC almost exactly fits description
+  - 70 - IOC mostly fits description
+  - 60 - IOC generally fits description
```

```suggestion
- defaultvalue: 'all'
  display: Sixgill Confidence
  name: confidence
  required: false
  type: 15
  options:
  - all
  - 90
  - 80
  - 70
  - 60
   additionalinfo: 'Ingest all IOCs=all, IOC exactly fits description=90, IOC almost exactly fits description=80, IOC mostly fits description=70, IOC generally fits description=60'
```

perhaps something like this? https://xsoar.pan.dev/docs/integrations/yaml-file#configuration (see additional-info section)